Dockerfile.ci

```diff
@@ -1,13 +1,16 @@
-FROM ruby:2.3.3-alpine
+FROM ruby:2.3.3
 
 # App home directory and app user can be injected through build params.
 ARG ARG_HOME=/app
 ARG ARG_USER=app
 
-RUN gem install bundler \
-    && apk add --no-cache build-base git \
-    && addgroup -S $ARG_USER \
-    && adduser -S -D -h /home/$ARG_USER -G $ARG_USER $ARG_USER
+RUN useradd -d /home/$ARG_USER -m --shell /bin/false --user-group $ARG_USER
```

Comment:
Do we rely on the ability to change the username and home directory?
If not, then I'm tempted to just remove the `ARG`s.